### ISSUE TITLE
Mail from muni.town

### ISSUE THREAD

**Issue by erlend-sh** [muni-town/weird#228]
As long as we’re relying on `muni.town` for signup emails we should clearly say on our signup page ‘you’ll be getting a mail from **muni.town**’ to reassure users that there’s nothing fishy going on when they receive an email from that alternate domain.

**Comment by zicklag** [comment]
It'd be good eventually to have like an official association on the Weird homepage / footer or something to show that Weird is affiliated with muni.town or something, too. Though maybe only after it has more of it's own presence.

**Comment by erlend-sh** [comment]
Good point! The footer is overdue for an update anyway (should be anchored to bottom and more subtle) so I’ll incorporate it in that ticket.
